frontend/server/config.ts

```diff
@@ -0,0 +1,48 @@
+const production = process.env.NODE_ENV === 'production'
+
+function get<T>(name: string, fallback: T, options = { requireInProduction: false }): T | string {
+    if (process.env[name]) {
+        return process.env[name]
+    }
+    if (fallback !== undefined && (!production || !options.requireInProduction)) {
+        return fallback
+    }
+    throw new Error(`Missing env var ${name}`)
+}
+
+const requiredInProduction = { requireInProduction: true }
+
+export class AgentConfig {
+    timeout: number
+
+    constructor(timeout = 8000) {
+        this.timeout = timeout
+    }
+}
+
+export interface ApiConfig {
+    url: string
+    timeout: {
+        response: number
+        deadline: number
+    }
+    agent: AgentConfig
+}
+
+export default {
+    staticResourceCacheDuration: 20,
+    apis: {
+        gdxDataSharePoc: {
+            url: get('GDX_DATA_SHARE_URL', 'http://127.0.0.1:63459', requiredInProduction),
```

Comment:
This is the port gdx-data-share-poc was running on when I last was running the site. Is there a way of pulling this through, or should I just add a comment?

Reply:
We should probably set this to 8080, because thats what it runs on in everything except for when running the docker compose through IDEA